package.json

```diff
@@ -5,14 +5,16 @@
   "scripts": {
     "gulp": "gulp",
     "bower": "bower",
-    "build": "gulp",
+    "build-prod": "cross-env BUILD_ENV=production gulp",
+    "build-dev": "cross-env BUILD_ENV=development gulp",
```

Comment:
just `build: 'gulp'`

Reply:
in that case you'll get null ENV, I prefer to make things obvious